ext/ldap/tests/ldap_deprectate_connect_with_two_parameters.phpt

```diff
@@ -0,0 +1,12 @@
+--TEST--
+ldap_connect() - With two parameters is marked as deprecated
+--CREDITS--
+Andreas Heigl <heiglandreas@php.net>
```

Comment:
Adding `--CREDITS--` sections to new test cases is discouraged, since there is already the commit giving implicit credit:
```suggestion
```